### PR TITLE
[FSDE-305] Added support of Chi Vue Search Input component

### PR DESCRIPTION
**Pull request by janibolkvadze**
@SergioQCL @mattnickles @jllr 

https://ctl.atlassian.net/browse/FSDE-305

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/3/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://08c44e7a769701df31ea1d654a06f92c5a227525-chi-pr.enterprise.ctl.io

File: src/chi-vue/src/views/SearchInputView.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<template>
+  <div id="search-input-view">
+    <div style="max-width: 20rem;">
```
Comment (SergioQCL) [inline src/chi-vue/src/views/SearchInputView.vue:3]: Maybe you can define a local class for this

File: src/chi-vue/src/components/search-input/SearchInput.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+@Component
+export default class SearchInput extends Vue {
+  @Prop() size?: SearchInputSizes;
+  @Prop() disabled?: boolean;
+  @Prop() name?: string;
+  @Prop() placeholder?: string;
+  @Prop() value?: string;
+
+  cleanButtonVisible = !!(this.$props.value && !this.$props.disabled);
+  inputValue = this.$props.value || '';
```
Comment (SergioQCL) [inline src/chi-vue/src/components/search-input/SearchInput.tsx:15]: Is this some kind of good practice? Can't these be called without using the $props object?

**Review by SergioQCL — COMMENTED** [review]
Just a couple of comments

File: src/chi-vue/src/views/SearchInputView.vue (hunk 1 above)
Comment (janibolkvadze) [reply src/chi-vue/src/views/SearchInputView.vue]: Done, thanks ;)

**Review by SergioQCL — APPROVED** [review]
ACK

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://09926951f5bb36161c476ea4295e2aa08b228604-chi-pr.enterprise.ctl.io

**Comment by janibolkvadze** [comment]
@jllr @mattnickles Can you review the PR, please?

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/10/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://4b26edf101fe8c027410b7f088b2e77c1a8a4728-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/12/. :x:

**Review by mattnickles — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/13/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-52/14/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b952650bae7159863daa43be039328087c63ef71-chi-pr.enterprise.ctl.io

**Pull request merged**